cloudferry_devlab/cloudferry_devlab/tests/config.py

```diff
@@ -70,6 +70,13 @@
 all_tenants_filter_filename = 'all_tenants_filter.yaml'
 """All tenants filter file."""
 
+rsc_map_filename = 'configs/resource_map.yaml'
+"""Resource Map file."""
+
+mapped_tenant_dict = {'tenant2': 'tenant-2'}
```

Comment:
Let's have some more descriptive tenant name instead of `tenant-2`, for example, `tenant2_mapped_in_resource_map_yaml`.